packages/e2e/cypress/integration/e2e/Canvas/Canvas.model.ts

```diff
@@ -0,0 +1,15 @@
+import { RootModel, selectors } from "../../../models"
+
+export class CanvasModel extends RootModel {
+  canvasPosition() {
+    return this.getCanvas().then(($el) => $el.css("transform"))
```

Comment:
Selectors to diffrent file

Reply:
but it's css props names, not selectors, should we move it in constants too?

Reply:
css styles Maybe or smth like this